revscoring/errors.py

```diff
@@ -75,6 +75,11 @@ def __init__(self, datasources, info=None, arg=None):
         super().__init__("Query failed ({0}:{1})"
                          .format(datasources, info))
 
+class UnexpectedContentType(DependencyError):
+    def __init__(self, text, arg=None):
+        super().__init__("Expected entity JSON "
+                         "but the following content can't be parsed: {}"
+                         .format(text))
```

Comment:
I think it would be best not to pre-fill the text of the error to be about JSON given that this is named as a general error for unexpected content.  Or you could make an "EntityContentTypeError" class that extends "UnexpectedContentType" and pre-fills this error message.

Reply:
Oh!  One more note.  `text` can be very long so it would be good to trim it down to the first N chars.  20 or 50 chars should be plenty for debugging.

Reply:
Good point, lemme know if the new version is better!